src/prommis/uky/costing/tests/test_ree_costing.py

```diff
@@ -5293,11 +5297,12 @@ def test_REE_costing_recovery(recovery_rate_units, expectation):
     m.fs.water.fix()
 
     m.fs.recovery_rate_per_year = pyo.Var(
-        initialize=39.3
-        * 0.8025
-        * 8
-        * 3
-        * 336,  # TREO (total rare earth oxide), 80.25% REE in REO
+        # initialize=39.3
+        # * 0.8025
+        # * 8
+        # * 3
+        # * 336,  # TREO (total rare earth oxide), 80.25% REE in REO
```

Comment:
If these aren't used, can we remove these lines?

Reply:
I wasn't sure how important the source of this number was. Happy to remove.